Source/WebInspectorUI/UserInterface/Views/OpenResourceDialog.js

```diff
@@ -339,6 +339,9 @@ WI.OpenResourceDialog = class OpenResourceDialog extends WI.Dialog
 
     _addResource(resource, suppressFilterUpdate)
     {
+        if (!resource)
+            return;
```

Comment:
id much rather us put this check at the few callsites that actually need it instead of assuming that _any_ caller could give `null`
```js
if (target.mainResource)
    this._addResource(target.mainResource);
```